conans/client/generators/cmake_common.py

```diff
@@ -519,49 +517,70 @@ def generate_targets_section(dependencies):
 endmacro()
 """
 
-cmake_macros = """
+
+def _conan_basic_setup_common(addtional_macros, cmake_multi=False):
+    output_dirs_section = """
+    if(NOT ARGUMENTS_NO_OUTPUT_DIRS)
```

Comment:
Maybe it should warn/info in the case the argument is defined, but is being ignored because it is cmake_multi?